test_scripts/WebEngine/WebEngineProjection/013_Registration_reject_app_has_no_permissions_to_use_WEB_VIEW_part_of_collection.lua

```diff
@@ -0,0 +1,46 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Check that App will be rejected with HMI type collection {"MEDIA", "NAVIGATION"} 
+-- when application does not have permission to use WEB_VIEW hmi type in policy table
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WebEngine device is connected to SDL
+-- 3. PT contains record for App1 
+
+-- Sequence:
+-- 1. Application1 try to register with WEB_VIEW appHMIType
+--  a. SDL reject registration of application (resultCode DISALLOWED, success:"false")
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local appHMITypeOnHMI = {"MEDIA", "NAVIGATION"}
```

Comment:
@YarikMamykin This is a collection of possible HMI types set in policy table, so I would propose to rename this variable:
```suggestion
local appHMITypeInPolicy = {"MEDIA", "NAVIGATION"}
```

Reply:
@aderiabin fixed in https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/32/commits/66e780b7fb2d744f4ac6fd6ee635b18b1bf5b68f